testgrid/config/config.yaml

```diff
@@ -0,0 +1,1193 @@
+test_group_default: &DEFAULT_TEST_GROUP
+  name: kubernetes-test-default
```

Comment:
Why include name/query here? Ideally you'd get an error if you inherited from the default and didn't override name/query.